lib/msal-node/docs/migration.md

```diff
@@ -1,171 +1,7 @@
-
-You can [declaratively configure](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/configuration.md) this object with JSON.
-
-#### Migrate from authority validation to known authorities
-
-ADAL has a flag to enable or disable authority validation.  Authority validation is a feature in ADAL/MSAL, that prevents your code from requesting tokens from a potentially malicious authority. ADAL retrieves a list of authorities known to Microsoft and validates the user provided authority against the retrieved set of authorities. Use of the flag is shown in the code snippet below.
-
-```js
-// With this flag you can turn on and off the authority validation
-// NOTE: The flag defaults to true
-var validateAuthority = true;
-
-var context = new AuthenticationContext(authorityUrl, validateAuthority);
-
-context.acquireTokenWithClientCredentials(resource, clientId, clientSecret, function(err, tokenResponse) {
-  if (err) {
-    console.log('well that didn\'t work: ' + err.stack);
-  } else {
-    console.log(tokenResponse);
-  }
-});
-```
-
-MSAL does not have a flag to disable authority validation, authorities are always validated. MSAL now compares your requested authority against a list of authorities known to Microsoft or a list of authorities you've specified in your configuration. Like illustrated in the code snippet below.
-
-```js
-// A user can include a list of know authorities to the config object to be used
-// during authority validation, as shown below.
-const msalConfig = {
-    auth: {
-        clientId: 'your_client_id',
-        authority: 'https://login.live.com',
-        knownAuthorities: ["login.live.com"],
-        protocolMode: "OIDC"
-    }
-};
-
-// Create msal application object
-const cca = new msal.ConfidentialClientApplication(msalConfig)
-```
-
-#### Logging
-To configure how MSAL logs it's activity you can specify the log level, callback and more in the configuration object passed during application instantiation as explained [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/configuration.md)
\ No newline at end of file
+While not strictly necessary, consider also migrating to Microsoft identity platform **v2.0 endpoint** if you haven't already: [Why update to Microsoft identity platform (v2.0)?](https://docs.microsoft.com/azure/active-directory/azuread-dev/azure-ad-endpoint-comparison)
```

Comment:
I believe the v2 endpoint is required for MSAL.

Reply:
MSAL Node supports both if I'm not mistaken :/ cc @sameerag

Reply:
The v2 endpoint is capable of returning v1 and v2 tokens, but I don't believe we should be telling folks they can still use the v1 endpoint.

Reply:
Sounds good then! Dropping this